src/main/java/com/iexec/common/utils/CategoriesUtils.java

```diff
@@ -0,0 +1,19 @@
+package com.iexec.common.utils;
```

Comment:
Maybe more in `chain` package instead of `utils` one if related to protocol ?

Reply:
Done: https://github.com/iExecBlockchainComputing/iexec-common/pull/355/commits/d3ad941a73ab4415cc5daaf82645bcd2035b5472 :)